examples/full.py

```diff
@@ -3,178 +3,230 @@
 import re
 
 from flask import Flask
-from slackify import (ACK, OK, Slackify, async_task, block_reply,
-                      request, respond, text_block, Slack)
-
+from slackify import (
+	ACK, OK, Slackify, async_task, block_reply,
+	request, respond, text_block, reply_text, Slack)
 
 app = Flask(__name__)
-slackify = Slackify(app=app)
+slackify = Slackify(app = app)
 cli = Slack(os.getenv('BOT_TOKEN'))
 
 
 @slackify.command
 def hello():
-    YES = 'yes'
-    NO = 'no'
-    yes_no_buttons_block = {
-        "type": "actions",
-        "elements": [
-            {
-                "type": "button",
-                "text": {
-                    "type": "plain_text",
-                    "emoji": True,
-                    "text": "Yes"
-                },
-                "style": "primary",
-                "value": "i_like_bots",
-                "action_id": YES
-            },
-            {
-                "type": "button",
-                "text": {
-                    "type": "plain_text",
-                    "emoji": True,
-                    "text": "No"
-                },
-                "style": "danger",
-                "value": "i_dont_like_bots",
-                "action_id": NO
-            }
-        ]
-    }
-    blocks = [
-        text_block('Do you like Bots?'),
-        yes_no_buttons_block
-    ]
-    return block_reply(blocks)
+	YES = 'yes'
+	NO = 'no'
+	yes_no_buttons_block = {
+		"type": "actions",
+		"elements": [
+			{
+				"type": "button",
+				"text": {
+					"type": "plain_text",
+					"emoji": True,
+					"text": "Yes"
+					},
+				"style": "primary",
+				"value": "i_like_bots",
+				"action_id": YES
+				},
+			{
+				"type": "button",
+				"text": {
+					"type": "plain_text",
+					"emoji": True,
+					"text": "No"
+					},
+				"style": "danger",
+				"value": "i_dont_like_bots",
+				"action_id": NO
+				}
+			]
+		}
+	blocks = [
+		text_block('Do you like Bots?'),
+		yes_no_buttons_block
+		]
+	return block_reply(blocks)
```

Comment:
I think this section might be modified by accident? Could you check if anything changed?
If it's not the case let's leave it as it was regarding spaces

Reply:
Yes, sorry about that. All the formatting issues should be fixed on the next push